seal.go

```diff
@@ -314,7 +339,43 @@ func SealKeyToTPM(tpm *TPMConnection, key []byte, keyPath string, params *KeyCre
 	}
 
 	succeeded = true
-	return goAuthKey.D.Bytes(), nil
+	return
```

Comment:
this function is a bit too long to use implicit return arguments,  I would have explicit:

return authKey, err  

here